### ISSUE TITLE
[A11y] Dashboard - Keyboard Navigation

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1844]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Cannot navigate to the "collapse" arrow that moves the dashboard links out of the way.

### Suggested Fix

Use a button instead of a div for this widget.  If a div must be used, it needs to use the role of "button" as well as all the required properties and JavaScript: https://www.w3.org/TR/wai-aria-practices-1.1/#button

**Comment by jechols** [comment]
QA: Verify that using the tab key alone, you can get to the "collapse" arrow:

![image](https://user-images.githubusercontent.com/2157974/129218836-613b29ce-b6bb-48e4-91c6-de539770a9ee.png)
